mw-webapp/src/converter/DayReportConverter.ts

```diff
@@ -6,35 +6,45 @@ import {DayReport as DayReportDTO} from "src/model/firebaseCollection/DayReport"
 import {JobDoneService} from "src/service/JobDoneService";
 import {PlanForNextPeriodService} from "src/service/PlanForNextPeriodService";
 import {CurrentProblemService} from "src/service/CurrentProblemService";
+import {MentorCommentService} from "src/service/MentorCommentService";
+import {MentorComment} from "src/model/businessModel/MentorComment";
 
-const jobsDoneRaw = await JobDoneService.onValueFromRealTimeDb();
-const plansForNextPeriodRaw = await PlanForNextPeriodService.onValueFromRealTimeDb();
-const problemsForCurrentPeriodRaw = await CurrentProblemService.onValueFromRealTimeDb();
+const jobsDoneRaw = await JobDoneService.getValueFromRealTimeDb();
+const plansForNextPeriodRaw = await PlanForNextPeriodService.getValueFromRealTimeDb();
+const problemsForCurrentPeriodRaw = await CurrentProblemService.getValueFromRealTimeDb();
+const mentorCommentsRaw = await MentorCommentService.getValueFromRealTimeDb();
 
 export const DayReportDTOToDayReportConverter = (dayReportRaw: DayReportDTO) => {
-  const jobsDone = dayReportRaw.jobsDone.map((item) => {
+  const jobsDone = dayReportRaw.jobsDone?.map((item) => {
     const jobDone: JobDone = jobsDoneRaw
       .find((elem) => elem.uuid === item) || jobsDoneRaw[0];
```

Comment:
make it simplier by moving to separate variable

Reply:
find is consuming operation, shold we optimise it? (for example use Map instaead of array)